internal/broker/broker.go

```diff
@@ -215,7 +214,43 @@ func (b Broker) LastOperation(ctx context.Context, instanceID, planID string) (d
 	return res, nil
 }
 
+func (b Broker) GetBinding(ctx context.Context, instanceID, bindingID string) (domain.GetBindingSpec, error) {
+	res := domain.GetBindingSpec{}
+
+	_, instance, err := b.getPlanInstance(ctx, "", instanceID)
```

Comment:
Do we even need to get the instance first?

If it's just to be able to call `crossplane.ServiceBinderFactory()` we should maybe work around that?

Reply:
hmm, scratch that. We really need the instance, to have the resource references to get to the secret for the binding.